### ISSUE TITLE
More Testing: The 1st testing assignment that references funfunfunction video API now seems to be paid and can't be used for free.

### ISSUE THREAD

**Issue by AnkitAdhikari** [TheOdinProject/curriculum#28379]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [x] Would you like to work on this issue?

### Describe your suggestion

The video of funfunfunction on Mocking uses a VAT API that (5:30), during the making of video the API seems free to use but now the API need to be purchased in order to use and people can't follow the video. 

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-more-testing

### (Optional) Discord Name

Xenon

### (Optional) Additional Comments

I love to help people and this is the challenge I faced and don't want other following Odin project go through it.

**Comment by mathdebate09** [comment]
Yeah the vat-api has moved to the credit card trial thing, but there are other free APIs like [AbstractAPI Pricing](https://www.abstractapi.com/api/vat-validation-rates-api#pricing) (provides 100 requests which should be enough to follow along)
This doesn't need to be mentioned, though I'd love to hear opinions, perhaps a caution box informing users?

**Comment by AnkitAdhikari** [comment]
Yes there should be some way of providing learners an alternate API like AbstractAPI

**Comment by JoshDevHub** [comment]
Thank you for making this issue @AnkitAdhikari 

Going to ping our JavaScript team and see what they want to do about this issue @TheOdinProject/javascript

**Comment by AnkitAdhikari** [comment]
since the assignment is little, I think we could add a simple alert pointing to some free API to follow along with the video. If what I mentioned is how we want to do it, then I would be happy to make changes to markdown, include an alternate API like AbstractAPI, and send a merge request. I want to get the feeling of contributing to an open-source project and want to make this first one.

**Comment by MaoShizhong** [comment]
In (https://github.com/TheOdinProject/curriculum/pull/27405#discussion_r1683131328), we've decided to replace the first assignment video in More Testing, i.e. the one this issue is about, so the replacement will complete this issue anyway. Going to open this up for contribution as the assignment box was left unchecked.

## Acceptance criteria

- [ ] `More Testing`'s first assignment step is reworded to link to https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s
- [ ] The paragraph at the end of `More Testing`'s assignment is indented by a total of 3 spaces (it's currently indented by only 2 spaces) so that it aligns with the 5th assignment step.
- [ ] The second assignment step in [`Testing Basics`](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics#assignment) has its last sentence removed (as it won't be relevant anymore).

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless a maintainer has assigned you.

**Comment by AnkitAdhikari** [comment]
Yes, I would love to work on this issue.

**Comment by MaoShizhong** [comment]
All yours then :)
